semconv/src/main/java/io/opentelemetry/semconv/NetworkAttributes.java

```diff
@@ -90,15 +91,13 @@ public static final class NetworkTransportValues {
     /** Unix domain socket */
     public static final String UNIX = "unix";
 
-    /** QUIC */
-    public static final String QUIC = "quic";
-
```

Comment:
oops, this has been around for a while 🙈

@jack-berg any thoughts? I'm guessing we need to wedge it back into the stable definition here somehow?

Reply:
https://github.com/open-telemetry/opentelemetry-js/pull/5690#issuecomment-2885209525 is the equivalent open question for the JS semconv package

Reply:
see https://github.com/open-telemetry/semantic-conventions/pull/2275!